packages/integration-test/src/stability.spec.ts

```diff
@@ -45,7 +46,16 @@ describe("stability check, assess a solution, reassess the solution, check all s
     const solutionPath: string = path.join(solutionFolderPath, "MvcMusicStore.sln");
     await addSolution(app, solutionPath);
     await app.client.refresh();
-    const results = await runner.runThroughSolution(solutionPath, "inplace", "net6.0", false, false);
+    const results = await runner.runThroughSolution(solutionPath, "inplace", "net6.0", false, false, {
+      apis: {
+        first: "td=ActionName",
+        last: "td=ViewResult",
+      },
+      sourceFiles: {
+        first: "=MvcMusicStore\\Controllers\\AccountController.cs",
```

Comment:
will this order always the same?

Reply:
only concern is if someone add new files to the project it might fails, but this check will work if the sorting always provides same result always.

Reply:
yeah I think the assumption is the test solution is pretty static, if the solution is changed then that can cause the test to fail, but then we can update the test, similar to if codelyzer starts detecting additional APIs the values reported will also change.